modules/react-components/src/components/accordion/segmented-accordion/segmented-accordion-title.tsx

```diff
@@ -117,9 +117,9 @@ export interface StrictSegmentedAccordionTitleActionInterface {
 /**
  * Segmented accordion title component.
  *
- * @param {SegmentedAccordionTitlePropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
- * @return {ReactElement}
+ * @returns
```

Comment:
Can we add what we are returning?

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3522/commits/59a9f310bf60598bccd0731fa9aa146bf7f412cc